src/main/resources/swagger/api-docs.yaml

```diff
@@ -202,61 +202,60 @@ paths:
-#        has permissions to read that fit the filter criteria.
-#      parameters:
-#        - name: namespace
-#         in: query
-#          description: Namespace of method.
-#          required: false
-#          type: string
-#        - name: name
-#          in: query
-#          description: Name of method.
-#          required: false
-#          type: string
-#        - name: snapshotId
-#          in: query
-#          description: Snapshot ID of method.
-#          required: false
-#          type: number
-#        - name: synopsis
-#          in: query
-#          description: Synopsis of method.
-#          required: false
-#          type: string
-#        - name: documentation
-#          in: query
-#          description: Documentation of method.
-#          required: false
-#          type: string
-#        - name: owner
-#          in: query
-#          description: Owner of method.
-#          required: false
-#          type: string
-#        - name: payload
-#          in: query
-#          description: Payload of method in WDL.
-#          required: false
-#          type: string
-#        - name: entityType
-#          in: query
-#          description: Type of Method - Task or Workflow
-#          required: false
-#          type: string
-#        - name: excludedFields
-#          in: query
-#          description: Fields to exclude in JSON response.
-#          required: false
-#          type: string
-#        - name: includedFields
-#          in: query
-#          description: Fields to include in JSON response
-#          required: false
-#          type: string
+      description: |
+        The configurations endpoint returns all configurations the requester
+        has permissions to read that fit the filter criteria.
+      parameters:
+        - name: namespace
+          in: query
+          description: Namespace of method.
+          required: false
+          type: string
+        - name: name
+          in: query
+          description: Name of method.
+          required: false
+          type: string
+        - name: snapshotId
+          in: query
+          description: Snapshot ID of method.
+          required: false
+          type: number
+        - name: synopsis
+          in: query
+          description: Synopsis of method.
+          required: false
+          type: string
+        - name: documentation
+          in: query
+          description: Documentation of method.
+          required: false
+          type: string
+        - name: owner
+          in: query
+          description: Owner of method.
+          required: false
+          type: string
+        - name: payload
+          in: query
+          description: Payload of method in WDL.
+          required: false
+          type: string
+        - name: entityType
+          in: query
+          description: Type of Method - Task or Workflow
+          required: false
+          type: string
+        - name: excludedFields
```

Comment:
This doesn't seem to work correctly - not sure if the issue is what orchestration is sending to agora or if agora just isn't honoring this query param.